sdk/python/tests/integration/registration/test_universal_registry.py

```diff
@@ -1343,3 +1350,138 @@ def validate_project_uuid(project_uuid, test_registry):
     assert len(test_registry.cached_registry_proto.project_metadata) == 1
     project_metadata = test_registry.cached_registry_proto.project_metadata[0]
     assert project_metadata.project_uuid == project_uuid
+
+
+@pytest.mark.integration
+@pytest.mark.parametrize("test_registry", all_fixtures)
+def test_apply_permission_success(test_registry):
```

Comment:
I don't see a test for READ_ONLINE or READ_OFFLINE permissions.

Reply:
@franciscojavierarceo There are tests for them here sdk/python/tests/integration/online_store/test_remote_online_store.py